.eslintrc

```diff
@@ -0,0 +1,21 @@
+{
+  "extends": [
+    "eslint:recommended",
+    "prettier"
+  ],
+  "parserOptions": {
+    "ecmaVersion": 6
+  },
+  "plugins": [
+    "markdown"
+  ],
+  "rules": {
+    "no-constant-condition": ["error", { "checkLoops": false }],
```

Comment:
I'm wondering if i should just inline overrides for this one where needed.

Reply:
maybe. I'll merge what we have now and we'll iterate